launch_test.go

```diff
@@ -833,6 +866,29 @@ func TestFetchParentBuildMeta(t *testing.T) {
 	}
 }
 
+func TestFetchParentEventMetaParseError(t *testing.T) {
+	oldMarshal := marshal
+	defer func() { marshal = oldMarshal }()
+
+	api := mockAPI(t, TestEventID, TestJobID, 0, "RUNNING")
+	api.eventFromID = func(eventID int) (screwdriver.Event, error) {
+		if eventID == TestParentEventID {
+			return screwdriver.Event(FakeEvent{ID: TestParentEventID}), nil
+		}
+		return screwdriver.Event(FakeEvent{ID: TestEventID, ParentEventID: TestParentEventID}), nil
+	}
+	marshal = func(v interface{}) (result []byte, err error) {
+		return []byte("test"), fmt.Errorf("Testing parsing parent event meta")
```

Comment:
You should return `nil` if you don't care about the `result`

Reply:
Fixed